src/clef.ts

```diff
@@ -143,7 +129,13 @@ export class Clef extends StaveModifier {
 
     this.setPosition(StaveModifierPosition.BEGIN);
     this.setType(type, size, annotation);
-    this.setWidth(Glyph.getWidth(this.clef.code, this.clef.point, `clef_${this.size}`));
+    this.setWidth(
+      Glyph.getWidth(
+        this.clef.code,
+        this.size == 'default' ? Tables.NOTATION_FONT_SCALE : (Tables.NOTATION_FONT_SCALE / 3) * 2,
```

Comment:
Can you add a comment here to explain why it is multiplied by 2/3 ?

Reply:
Done in `Clef.getPoint`